### ISSUE TITLE
Update to camunda v7.15.0

### ISSUE THREAD

**Issue by joaopalma5** [paulbares/camunda-notification#1]
Update code to 7.15.0

**Comment by joaopalma5** [comment]
I create this checkpoint: https://github.com/joaopalma5/camunda-notification-1

I Have this errors, during the Unit tests
```
[ERROR] Failures: 
[ERROR]   TestApprovalWorkflowRegistrar.testCheckPublishHappensOnlyOnce:123 
Actual and expected have the same elements but not in the same order, at index 0 actual element was:
  <"0">
whereas expected element was:
  <"2">

[ERROR]   TestApprovalWorkflowRegistrar.testNotificationWithOnlyOneSubscriber:69 
Actual and expected have the same elements but not in the same order, at index 0 actual element was:
  <"0">
whereas expected element was:
  <"2">

[ERROR]   TestApprovalWorkflowRegistrar.testSeveralSubscribersSameUser:153 
Actual and expected have the same elements but not in the same order, at index 0 actual element was:
  <"0">
whereas expected element was:
  <"2">

[INFO]
[ERROR] Tests run: 16, Failures: 3, Errors: 0, Skipped: 0
```

**Comment by paulbares** [comment]
@joaopalma5 issue was not related to camunda 7.15.0 but to incorrect test setup. I fixed it and update to camunda 7.15.0: 2994e82a15cc7eb5d87da752e98f161c0fd69964
